packages/prelude/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@most/prelude",
+  "version": "1.5.2",
```

Comment:
Hmmm, I didn't think about versioning implications.  Anyone know the right way to handle this in lerna, given that we're not currently using `--independent`?